colaraz/cms/templates/base.html

```diff
@@ -242,9 +242,14 @@
       </script>
     </%block>
       <script type="text/javascript">
+        // Colaraz Notifications API variables
         var colarazNotificationsFetchingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'fetch'})}";
         var colarazNotificationsMarkingUrl = "${reverse('colaraz_features:notifications_handler', kwargs={'api_method': 'mark'})}";
-        var notificationsRefreshTime = ${getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_RATE_MILLISECONDS', 30000)};
+        var colarazNotificationsRefreshTime = ${getattr(settings, 'COLARAZ_NOTIFICATIONS', {}).get('REFRESH_RATE_MILLISECONDS', 30000)};
+        // Colaraz Job Alerts API variables
+        var colarazJobAlertsFetchingUrl = "${reverse('colaraz_features:job_alerts_handler', kwargs={'api_method': 'fetch'})}";
+        var colarazJobAlertsMarkingUrl = "${reverse('colaraz_features:job_alerts_handler', kwargs={'api_method': 'mark'})}";
+        var colarazJobAlertsRefreshTime = ${ getattr(settings, 'COLARAZ_JOB_ALERTS', {}).get('REFRESH_TIME_MILLISECONDS', 30000)};
```

Comment:
Since you are using `RATE` word for the `COLARAZ_NOTIFICATIONS`, it would be better to use that here too. or the opposite.
```suggestion
        var colarazJobAlertsRefreshTime = ${ getattr(settings, 'COLARAZ_JOB_ALERTS', {}).get('REFRESH_RATE_MILLISECONDS', 30000)};
```

Reply:
good catch, updated.